subsys/mgmt/mcumgr/grp/img_mgmt_client/src/img_mgmt_client.c

```diff
@@ -266,7 +266,8 @@ static size_t upload_message_header_size(struct img_gr_upload *upload_state)
 	} else {
 		map_count = 10;
 	}
-	temp_data = NULL;
+
+	memset(temp_data, 0, sizeof(temp_data));
```

Comment:
I don't really know what you are doing? Remove this and change line 258 to:
```
	uint8_t temp_data = 0;
```

Reply:
sorry, I just not too use to git.